lms/models/region.py

```diff
@@ -18,25 +18,21 @@ class Region:
 class Regions:
     """A collection of all the regions as an enum like object."""
 
-    # As region objects are frozen, these can be directly compared like enums
```

Comment:
I'm not sure what this comment is getting at. Dataclasses can be directly compared (with `obj1 == obj2`) by default even without `frozen=True`.

What `frozen=True` does is make them _hashable_, but that doesn't appear to be relevant here.